test/e2e/docker-compose-liquidation.yml

```diff
@@ -0,0 +1,54 @@
+version: '3.9'
+services:
+  synpress:
+    extends:
+      file: docker-compose-base.yml
+      service: synpress
+    depends_on:
+      - display
+      - video
+      - agd
+    command: >
+      bash -c 'echo -n "======> local noVNC URL: http://localhost:8080/vnc.html?autoconnect=true " &&
+      yarn wait-on http://display:8080 &&
+      echo -n "======> remote noVNC URL: " &&
+      curl -s ngrok:4040/api/tunnels | jq -r .tunnels[0].public_url &&
+      nginx &&
+      npx start-server-and-test "yarn dev" http-get://localhost:5173 "yarn test:e2e --spec test/e2e/specs/liquidation.spec.js"'
```

Comment:
`npx` will hit the external network to fetch. why not make it a devDependency?

Reply:
It is a devDependency:
https://github.com/rabi-siddique/rs-dapp-inter/blob/rs-initial-test-cases-for-liquidation/package.json#L78

Reply:
Ah then no need for npx. 'npm run'